src/21-pubsub/index.md

```diff
@@ -0,0 +1,105 @@
+---
+# this is the final blog post's id (used in the directory)
+# this id is part of the url, and should only contain:
+#   letters, numbers, dashes.
+id: 21-pubsub
+
+breadcrumbs:
+  - {name: "Take a look at pubsub on ipfs!", link: "./" }
+
+# the date here should be set to the final publication date,
+# on the day it is published.
+date: 2016-11-09
+
+# this is the Title
+title: Take a look at pubsub on ipfs!
+
+# this is the name of the main author(s)
+author: Jeromy Johnson
+
+# technical details required for the software, don't change these.
+baseurl: ..
+template: tmpl/layouts/post.html
+collection: posts
+---
+
+We are very excited to announce that we have recently merged a simple,
+experimental pubsub implementation.  While the implementation is experimental,
```

Comment:
Structure the intro like so:

> \<some hook, like: To make IPFS fast for data centers, local area networks, and large p2p applications, we need pubsub.> \<Explanation of what pubsub is here, with links to wikipedia and papers, and why it's valuable>
> 
> \<paragraph explaining that we did a review of papers. maybe say we're hoping to do a write up about it, but that we're busy implementing so no ETA.>
>
> \<paragraph with a bit of roadmap, like (a) lit review, (b) think of interfaces and proposals, (c) libp2p interfaces, (d) MVP implementation (floodsub), (e) use in orbit-db for orbit, (f) build network benchmarking tooling, (g) propose better implementations, (h) implement, test, and benchmark them. (i) pubsub is ready \o/>
> 
> We recently merged a simple, experimental pubsub implementation...

Important because:
- clarifies why this is important.
- clarifies the strategy in general (roadmap)
- it clarifies where we are in the roadmap
- clarifies what this thing is (dumb first stab)
- points to other ongoing work
- clarifies what's next, invite people to get involved.

Also, downplays the "very excited". remember that floodsub is absolutely terrible for bandwidth, and totally brain damaged. any serious p2p person would point and laugh. as do we.